ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -261,6 +277,16 @@ impl<'a> TraceBuilder<'a> {
                 let const_idx = self.jit_mod.const_idx(&jit_const)?;
                 jit_ir::Operand::Const(const_idx)
             }
+            aot_ir::Operand::Arg(arg_idx) => {
+                // The `Arg` operand is an index for the argument of this function. We can use it
+                // to lookup the AOT argument being passed into this function from the call
+                // instruction. We then lookup what JIT instruction the AOT argument maps to and
+                // return this.
+                // Unwrap is safe since an `Arg` means we are currently inlining a function.
+                // FIXME: Is the above correct? What about args in the control point frame?
```

Comment:
Good question. Those will be trace inputs...

Reply:
Yes, but we might have to map them in a similar way to the trace inputs.